iidm/iidm-api/src/main/java/com/powsybl/iidm/network/Network.java

```diff
@@ -70,6 +70,11 @@
          * Get the switch count.
          */
         int getSwitchCount();
+
+        /**
+         * Get a Bus.
+         */
+        Bus getBus(String id);
```

Comment:
Should be default when adding a new interface, otherwise there is compile issue for sub-class in others project.